.stack-to-nix.cache

```diff
@@ -29,3 +29,28 @@ https://github.com/input-output-hk/ouroboros-network 408f3a3fc8be601075ee2bacdb7
 https://github.com/input-output-hk/ouroboros-network 408f3a3fc8be601075ee2bacdb7b7b9ebac9d2c2 typed-protocols 0hc8p7h60wq4xp43pl7rqn94lra7gy51z3h6jjqvy6lj4bcfrp65 typed-protocols typed-protocols.nix
```

Comment:
I really do not think this file should even be in the repo.

Reply:
I agree. I think it goes away with the move to the updated haskell nix infrastructure right?